test/functional/wallet_multiwallet.py

```diff
@@ -80,7 +81,9 @@ def wallet_file(name):
         # rename wallet.dat to make sure plain wallet file paths (as opposed to
         # directory paths) can be loaded
         # create another dummy wallet for use in testing backups later
-        self.start_node(0, ["-nowallet", "-wallet=empty", "-wallet=plain"])
+        self.start_node(0)
+        node.createwallet("empty", descriptors=False)
```

Comment:
When possible, can we please avoid forcing legacy wallets? Not all of these newly created wallets need to be legacy.

Reply:
re: https://github.com/bitcoin/bitcoin/pull/20186#discussion_r512266002

> When possible, can we please avoid forcing legacy wallets? Not all of these newly created wallets need to be legacy.

Obviously yes, but these wallets were legacy before this PR and they are legacy after this PR. This PR isn't trying to change wallet types, and I wasn't trying to make things more difficult for myself by introducing unnecessary changes at the same time as fixing all of these tests. But I can make the change if an update is needed. I'm reviewing #18788 which updates tests to use descriptor wallets, and think that PR is another good place to take up this issue.

Reply:
Hmm, then `wallet_multiwallet.py --descriptors` is currently very misleading as it doesn't actually test multiwallet descriptor wallets.

Reply:
re: https://github.com/bitcoin/bitcoin/pull/20186#discussion_r512293907

> Hmm, then `wallet_multiwallet.py --descriptors` is currently very misleading as it doesn't actually test multiwallet descriptor wallets.

I'm not seeing that. There are createwallet calls that don't set descriptors=False, including the one that creates the default wallet. I know I definitely made updates to this test to deal with wallet.sqlite issues that happened only with descriptor wallets, but maybe not all of these changes survived the reverted naming. Going forward other createwallet calls can be updated and I'm happy to do it here if there need to be updates here, or I think #18788 could be another logical place to switch more legacy wallets to descriptor wallets